sections/futures/Trade/TradeBalance.tsx

```diff
@@ -1,55 +1,97 @@
-import { memo, useState } from 'react';
+import { memo, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
 
 import CaretDownIcon from 'assets/svg/app/caret-down-gray.svg';
 import Button from 'components/Button';
-import { FlexDivRowCentered } from 'components/layout/flex';
+import { FlexDivCol, FlexDivRowCentered } from 'components/layout/flex';
 import { Body, NumericValue } from 'components/Text';
+import { MIN_MARGIN_AMOUNT } from 'constants/futures';
 import { setOpenModal } from 'state/app/reducer';
+import { selectShowModal } from 'state/app/selectors';
+import { selectSusdBalance } from 'state/balances/selectors';
 import {
 	selectAvailableMargin,
 	selectFuturesType,
 	selectIdleMargin,
 	selectWithdrawableMargin,
 } from 'state/futures/selectors';
 import { useAppDispatch, useAppSelector } from 'state/hooks';
-import { formatDollars } from 'utils/formatters/number';
+import { formatDollars, zeroBN } from 'utils/formatters/number';
 
 import CrossMarginInfoBox from '../TradeCrossMargin/CrossMarginInfoBox';
+import TransferIsolatedMarginModal from './TransferIsolatedMarginModal';
 
 type TradeBalanceProps = {
 	isMobile?: boolean;
 };
 
 const TradeBalance: React.FC<TradeBalanceProps> = memo(({ isMobile = false }) => {
+	const { t } = useTranslation();
 	const dispatch = useAppDispatch();
+
 	const idleMargin = useAppSelector(selectIdleMargin);
 	const accountType = useAppSelector(selectFuturesType);
 	const availableIsolatedMargin = useAppSelector(selectAvailableMargin);
 	const withdrawable = useAppSelector(selectWithdrawableMargin);
+	const susdBalance = useAppSelector(selectSusdBalance);
```

Comment:
I don't think we need this, the check should just be based on `idleMargin` which will include wallet balance, idle margin in markets and smart margin account balance.

Reply:
Thanks for pointing this out. Updated.